oss_build.sh

```diff
@@ -87,17 +87,39 @@ for python_version in $PYTHON_VERSIONS; do
     bazel clean
   fi
 
-  if [ "$python_version" = "3.6" ]; then
-    export PYTHON_BIN_PATH=/usr/bin/python3.6 && export PYTHON_LIB_PATH=/usr/local/lib/python3.6/dist-packages
-  elif [ "$python_version" = "3.7" ]; then
-    export PYTHON_BIN_PATH=/usr/local/bin/python3.7 && export PYTHON_LIB_PATH=/usr/local/lib/python3.7/dist-packages
-    ABI=cp37
-  elif [ "$python_version" = "3.8" ]; then
-    export PYTHON_BIN_PATH=/usr/bin/python3.8 && export PYTHON_LIB_PATH=/usr/local/lib/python3.8/dist-packages
-    ABI=cp38
+  if [ "$(uname)" = "Darwin" ]; then
+    if [ "$python_version" = "3.6" ]; then
+      export PYTHON_BIN_PATH=python3.6
```

Comment:
why not absolute paths?  relative seems dangerous.

Reply:
On macOS, people may have more than one installed python3. I think `PYTHON_BIN_PATH=python3.6` will use the configured one.
I test the changes on `$HOME/opt/anaconda3/bin/python3.6`